src/Libraries/DSCPython/CPythonEvaluator.cs

```diff
@@ -471,11 +495,17 @@ public static DataMarshaler InputMarshaler
             }
         }
 
+        /// <summary>
+        ///     Data Marshaler for all data coming into a Python node.
+        /// </summary>
+        [SupressImportIntoVM]
+        public static DataMarshaler InputMarshaler => Instance.InputDataMarshaler as DataMarshaler;
+
         /// <summary>
         ///     Data Marshaler for all data coming out of a Python node.
         /// </summary>
         [SupressImportIntoVM]
-        public static DataMarshaler OutputMarshaler
+        public override object OutputDataMarshaler
```

Comment:
It needs to replace an existing public API
PythonEngine.OutputDataMarshaler will be used instead of the already existing public API CPythonEvaluator.OutputMarshaler